### PR TITLE
feat: add an isDisabled function to identify whether elements are disabled or not

### PR DESCRIPTION
**Pull request by MatanBobi**
This was discussed in https://github.com/testing-library/dom-testing-library/pull/1221#discussion_r1147913695

I wasn't sure where's the right place to put this one, please let me know if there's a better place for this and I'll change (I thought about creating a new file but didn't want to make a mess).

Thanks!

### CODE REVIEW

**Review by eps1lon — CHANGES_REQUESTED** [review]
These seems very much like an implementation detail. I assumed you meant to add a method that checks if an element is considered disabled in the a11y tree.

But this is too low level to be considered for a public API.

**Comment by MatanBobi** [comment]
> These seems very much like an implementation detail. I assumed you meant to add a method that checks if an element is considered disabled in the a11y tree.
> 
> But this is too low level to be considered for a public API.

That makes a lot more sense.
Will fix this, thanks.

File: sources/__tests__/is-disabled.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { isDisabled } from "../is-disabled";
+import { cleanup, renderIntoDocument } from "./helpers/test-utils";
+
+describe("isInaccessible", () => {
```
Comment (eps1lon) [inline sources/__tests__/is-disabled.js:4]:
```suggestion
describe("isDisabled", () => {
```

**Review by eps1lon — APPROVED** [review]
Thanks!

**Pull request merged**